code/controllers/subsystem/processing/resinshaping.dm

```diff
@@ -0,0 +1,44 @@
+#define QUICKBUILD_STRUCTURES_PER_XENO 300
+
+SUBSYSTEM_DEF(resinshaping)
+	name = "Resin Shaping"
+	flags = SS_NO_FIRE | SS_NO_INIT
+	var/list/xeno_builds_counter = list()
+	var/total_structures_built = 0
+	var/total_structures_refunded = 0
+	var/quickbuilds_per_xenomorph = QUICKBUILD_STRUCTURES_PER_XENO
```

Comment:
Why is that a var

Reply:
so it could be auto-balanced if admins deem theres not enough xenos or the auto-balancer steps in

Reply:
Admin shouldnt interfere with that.

Also, the amount of maze being proportional to the amount of builder/player is perfectly fine